led_strip/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ include($ENV{IDF_PATH}/tools/cmake/version.cmake)
 set(srcs "src/led_strip_api.c")
 
 if(CONFIG_SOC_RMT_SUPPORTED)
-    list(APPEND srcs "src/led_strip_rmt_dev.c" "src/led_strip_rmt_encoder.c")
+    if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "5.0")
+        list(APPEND srcs "src/led_strip_rmt_dev_idf5.c" "src/led_strip_rmt_encoder.c")
```

Comment:
nitpick, just the `idf5` sounds weird, I'm sure we will continue with the current RMT driver implementation in esp-idf v6/v7.